library/proc_macro/src/lib.rs

```diff
@@ -918,32 +918,29 @@ impl !Send for Punct {}
 #[stable(feature = "proc_macro_lib2", since = "1.29.0")]
 impl !Sync for Punct {}
 
-/// Indicates whether a `Punct` token can join with the following token
-/// to form a multi-character operator.
+/// Indicates whether a `Punct` token can join with the following token to form a new token.
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[stable(feature = "proc_macro_lib2", since = "1.29.0")]
 pub enum Spacing {
-    /// A `Punct` token can join with the following token to form a multi-character operator.
+    /// A `Punct` token can join with the following token to form a new token.
     ///
-    /// In token streams constructed using proc macro interfaces `Joint` punctuation tokens can be
-    /// followed by any other tokens. \
```

Comment:
Well, I tried to insert the line breaks for the docs to look good...
But yeah, it's not something commonly used in doc comments, I think.